src/main/java/bio/terra/controller/ApiValidationExceptionHandler.java

```diff
@@ -39,5 +41,15 @@
 
         return new ResponseEntity<>(errorModel, HttpStatus.BAD_REQUEST);
     }
+
```

Comment:
Hmm... should this also be in ControllerUtils?

Reply:
Do you see or can you think of other opportunities for us to use this? My thinking is that since this is our global exception handler for converting exceptions to `ErrorModel`s we wouldn't need this in other places.

Reply:
Maybe I misunderstood. I thought this was a general way for validators to format errors. I was guessing with multiple controllers we would have validators in different places.